frameworks/Java/smart-socket/src/main/java/org/smartboot/http/JSON.java

```diff
@@ -2,12 +2,22 @@
 
 import com.alibaba.fastjson.JSONObject;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * @author 三刀
  * @version V1.0 , 2018/8/12
  */
 public class JSON {
-    public static byte[] toJson(Object obj) {
-        return JSONObject.toJSONBytes(obj);
+    private static Map<String, byte[]> cacheMap = new HashMap<>();
```

Comment:
Hi @smthing -

We're not going to allow caching of JSON serialization. We'll make it a point to be more specific about this in the rules.